fetcher/handlers/l0_fetcher.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import subprocess
+import tempfile
+from typing import Any, List, Dict
+
+import boto3
+
+
+BotoClient = Any
+Event = Dict[str, Any]
+
+FTP_PATH = "/pub/OPS/TM/"
+
+
+def get_rclone_config_path(
+    ssm_client: BotoClient,
+    rclone_config_ssm_name: str
+) -> str:
+
+    rclone_config = ssm_client.get_parameter(
+        Name=rclone_config_ssm_name, WithDecryption=True
+    )["Parameter"]["Value"]
+
+    f = tempfile.NamedTemporaryFile(buffering=0, delete=False)
+    f.write(rclone_config.encode())
+
+    return f.name
+
+
+def format_command(config_path: str, bucket: str, full_sync: bool) -> List[str]:
+    cmd = [
+        "rclone",
+        "--config",
+        config_path,
+        "sync",
+        f"FTP:{FTP_PATH}",
+        f"S3:{bucket}",
+    ]
+
+    if not full_sync:
+        cmd += ["--update", "--use-server-modtime", "--max-age", "7d"]
+
+    return cmd
+
+
+def lambda_handler(event: Event, _):
```

Comment:
are you sure that the second argument doesn't have to be named `context`?

Reply:
I think so, I've tried deploying and testing the lambda and no problems detected